# joint name
string joint_name

# velocity threshold
float64 velocity_threshold

# torque threshold
float64 torque_threshold

# commanded joint velocity
float64 joint_velocity

# commanded joint torque
float64 joint_torque
